<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphwindow</class>
 <widget class="QDialog" name="graphwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_df">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>D( f ) =</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="widget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">widget-&gt;setBackground(QBrush(Qt::transponse))</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_emax">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dmax">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dmin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_print_graph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ef">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>E( f ) =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_emin">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clear">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dmin_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
   <property name="text">
    <string>-5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dmax_1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emin_1">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
   <property name="text">
    <string>-5</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emax_1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QFrame</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
